code/game/objects/random/food.dm

```diff
@@ -1,8 +1,26 @@
-/obj/random/rotfood
+/obj/random/junkfood
+	name = "random junkfood"
+	icon_state = "food-red"
+
+/obj/random/junkfood/item_to_spawn()
+	return pickweight(list(/obj/item/weapon/reagent_containers/food/snacks/chips = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/candy = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/unajerky = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/cheesiehonkers = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/tastybread = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/no_raisin = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/spacetwinkie = 3,
+				/obj/item/weapon/reagent_containers/food/drinks/dry_ramen = 2,
+				/obj/item/weapon/reagent_containers/food/snacks/hotdog = 1,
+				/obj/item/weapon/reagent_containers/food/snacks/liquidfood = 2,
+				/obj/item/weapon/reagent_containers/food/snacks/pie = 1,
+				/obj/item/weapon/reagent_containers/food/snacks/sandwich = 1))
+
+/obj/random/junkfood/rotten
 	name = "random spoiled food"
 	icon_state = "food-red"
 
-/obj/random/rotfood/item_to_spawn()
+/obj/random/junkfood/rotten/item_to_spawn()
```

Comment:
rotten food doesn't really need its own seperate list, does it? 

Wouldn't it be better to use the junkfood list and then adjust things in postspawn to add toxins or somesuch?